### PR TITLE
schema create

### PR DESCRIPTION
**Pull request by kumuthac23**

### CODE REVIEW

File: database/models/admission.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const AdmissionSchema = new mongoose.Schema({
+  name: String,
+  Aadhar: Number,
```
Comment (praveensumathi) [inline database/models/admission.js:5]: change the name to camelcase

File: database/models/admission.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  nationality: String,
+  dateOfBirth: String,
+  religion: String,
+  fatherName: String,
+  motherName: String,
+  address: String,
+  pnoneNumber: Number,
+});
+
+const AdmissionModel = mongoose.model("admission", AdmissionSchema);
```
Comment (praveensumathi) [inline database/models/admission.js:18]: change the model name to "admissions"

File: database/models/career.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+const CareerSchema = new mongoose.Schema({
+  name: String,
+  gender: String,
+  DOB: String,
```
Comment (praveensumathi) [inline database/models/career.js:6]: give full name like dateOfBirth, also the type should be date format

File: database/models/career.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+const CareerSchema = new mongoose.Schema({
+  name: String,
+  gender: String,
```
Comment (praveensumathi) [inline database/models/career.js:5]: can we do the enums type for the gender("Male","Female","Other")

File: database/models/career.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  DOB: String,
+  contactNumber: Number,
+  address: String,
+  yearOfExperience: Number,
+  presentlyWorking: String,
+  marriedStatus: String,
+  emailId: String,
```
Comment (praveensumathi) [inline database/models/career.js:12]: change the emailId to email

File: database/models/career.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  educationualification: String,
```
Comment (praveensumathi) [inline database/models/career.js:13]: spelling mistake

File: database/models/career.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+});
+
+const CareerModel = mongoose.model("career", CareerSchema);
```
Comment (praveensumathi) [inline database/models/career.js:16]: change the "career" to "careers"

File: database/models/contact.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+const ContactSchema = new mongoose.Schema({
+  childName: String,
+  parentName: String,
+  phoneNumber: Number,
+  email: String,
+  message: String,
+});
+
+const ContactModel = mongoose.model("Contact", ContactSchema);
```
Comment (praveensumathi) [inline database/models/contact.js:11]: change the "Contact" to "Enquires"

File: database/models/contact.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+
+module.exports = ContactModel;
```
Comment (praveensumathi) [inline database/models/contact.js:13]: change the Contact Model to "EnquiresModel"

File: database/models/contact.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+
+const ContactSchema = new mongoose.Schema({
```
Comment (praveensumathi) [inline database/models/contact.js:3]: change the "ContactSchema" to "EnquirySchema"

File: database/models/gallery.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const GallerySchema = new mongoose.Schema({
+  images: [
```
Comment (praveensumathi) [inline database/models/gallery.js:4]: change the images to imageUrl

File: database/models/gallery.js [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+    {
+      type: String,
+    },
+  ],
+  media: { type: mongoose.Schema.Types.ObjectId, ref: MediaModel },
```
Comment (praveensumathi) [inline database/models/gallery.js:9]: change the media to mediaId

File: database/models/media.js [hunk 13]
```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose");
+
+const MediaSchema = new mongoose.Schema({
+  title: String,
+  description: String,
+  type: Number,
+});
+EventsSchema.set('timestamps', true);
```
Comment (praveensumathi) [inline database/models/media.js:8]: change the schema name

File: database/models/media.js [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+
+const MediaModel = mongoose.model("events", MediaSchema);
```
Comment (praveensumathi) [inline database/models/media.js:10]: change "events" to "media"

File: database/models/admission.js (hunk 1 above)
Comment (kumuthac23) [reply database/models/admission.js:5]: changed

File: database/models/admission.js (hunk 2 above)
Comment (kumuthac23) [reply database/models/admission.js:18]: changed

File: database/models/career.js (hunk 3 above)
Comment (kumuthac23) [reply database/models/career.js:6]: changed

File: database/models/career.js (hunk 4 above)
Comment (kumuthac23) [reply database/models/career.js:5]: changed

File: database/models/career.js (hunk 5 above)
Comment (kumuthac23) [reply database/models/career.js:12]: changed

File: database/models/contact.js (hunk 10 above)
Comment (kumuthac23) [reply database/models/contact.js:3]: changed

File: database/models/gallery.js (hunk 11 above)
Comment (kumuthac23) [reply database/models/gallery.js:4]: changed

File: database/models/gallery.js (hunk 12 above)
Comment (kumuthac23) [reply database/models/gallery.js:9]: changed

File: database/models/media.js (hunk 13 above)
Comment (kumuthac23) [reply database/models/media.js:8]: remove EventsSchema

File: database/models/media.js (hunk 14 above)
Comment (kumuthac23) [reply database/models/media.js:10]: changed to media

**Review by praveensumathi — APPROVED** [review]

**Pull request merged**